src/items.cpp

```diff
@@ -1069,12 +1058,12 @@ SDL_Surface* itemSprite(Item* item)
 		node_t* node = list_Node(&items[item->type].surfaces, item->appearance % items[item->type].variations);
 		if ( !node )
 		{
-			return NULL;
+			return nullptr;
 		}
-		SDL_Surface** surface = (SDL_Surface**)node->element;
+		auto** surface = static_cast<SDL_Surface**>(node->element);
```

Comment:
unnecessary auto, recommend leave as SDL_Surface**

Reply:
Isn't it the double explicit typing that's unnecessary though?
For example, if you needed to change this from being `SDL_Surface**` to some other type, you now need to change both instances.
I'm highly for explicit typing in almost every single case, but I do find that this case using auto is better, as it reduces overhead costs in development type if this ever needs to change.

Reply:
Fixed in 3010621c